tpp/src/main/TokenClient.js

```diff
@@ -58,14 +58,31 @@ export class TokenClient extends Core {
      * @param  alias - alias for the member
      * @param  CryptoEngine - engine to use for key creation and storage
      * @param  realmId - (optional) member id of the Member to which this new member will belong
+     * @param  tppType - (optional) TPP type of the Member, default is Type 1
      * @return Promise of created Member
      */
     createMember(
         alias: ?Alias,
         CryptoEngine: Class<KeyStoreCryptoEngine>,
-        realmId?: string
+        realmId?: string,
+        tppType?: Number
     ): Promise<Member> {
-        return super.createMemberCore(alias, CryptoEngine, Member, 'BUSINESS', undefined, realmId);
+        let memberType = 'BUSINESS';
+        if (typeof tppType !== 'undefined' ) {
+            switch(tppType) {
+            case 2:
```

Comment:
I would not use "Type" name in SDK. It is internal terminology. The external one is reflected in lib-proto CreateMemberType.
We discussed public naming for quite a long time with the product team and we should use this official naming everywhere: 
T1 = BUSINESS - a customer of Token. Token is the regulated TPP in this case
T2 = LICENSED_TPP - a customer of Token but uses its own license. Token is a TSP for the business in this case.